src/reference/asciidoc/kafka.adoc

```diff
@@ -1171,6 +1171,22 @@ public KafkaTemplate<String, String> myReplyingTemplate() {
 }
 ----
 
+IMPORTANT: If the listener method returns `Message<?>` or `Collection<Messge<?>>` the listener method is responsible for setting up the message headers for the reply; for example, when handling a request from a `ReplyingKafkaTemplate`, you might do the following:
```

Comment:
When `if`, `when` etc is in the first part of the sentence, it must be separated with a comma : https://www.ego4u.com/en/cram-up/writing/comma?10 😉 

I mean in this case we need comma before `the listener method`.

Reply:
Hehe - thanks, yes, please fix on merge (unless you want me to do something with your other comment).